promgen/static/js/promgen.vue.js

```diff
@@ -180,7 +197,16 @@ app.component('silence-form', {
                     }
                 });
         },
-    }
+        hideModal: function (modalId) {
+            var modal = $('#' + modalId);
```

Comment:
```suggestion
        hideModal() {
            const modal = $('#silenceModal');
```

Reply:
The method does not need to be "configurable", and also, we don't need to write `function`, so the code becomes shorter.
Also, as a good practice, always use `const` when possible.